messages/md.convert.json

```diff
@@ -0,0 +1,28 @@
+{
+  "description": "convert metadata from the Metadata API format into the source format\nConverts metadata retrieved via Metadata API into the source format used in Salesforce DX projects.\nTo use Salesforce CLI to work with components that you retrieved via Metadata API, first convert your files from the metadata format to the source format using \"sfdx force:mdapi:convert\".\n\nTo convert files from the source format back to the metadata format, so that you can deploy them using \"sfdx force:mdapi:deploy\", run \"sfdx force:source:convert\".",
+  "examples": [
+    "$ sfdx force:mdapi:convert -r path/to/metadata",
+    "$ sfdx force:mdapi:convert -r path/to/metadata -d path/to/outputdir"
+  ],
+  "flags": {
+    "rootdir": "the root directory containing the Metadata API–formatted metadata",
+    "outputdir": "the output directory to store the source–formatted files",
+    "manifest": "file path to manifest (package.xml) of metadata types to convert.",
+    "metadatapath": "comma-separated list of metadata file paths to convert",
+    "metadata": "comma-separated list of metadata component names to convert"
+  },
+  "flagsLong": {
+    "rootdir": "The root directory that contains the metadata you retrieved using Metadata API.",
+    "outputdir": "The directory to store your files in after they’re converted to the source format. Can be an absolute or relative path.",
+    "manifest": [
+      "The complete path to the manifest (package.xml) file that specifies the metadata types to convert.",
+      "If you specify this parameter, don’t specify --metadata or --sourcepath."
+    ],
+    "metadatapath": [
+      "A comma-separated list of paths to the local metadata files to convert. The supplied paths can be to a single file (in which case the operation is applied to only one file) or to a folder (in which case the operation is applied to all metadata types in the directory and its sub-directories).",
+      "If you specify this parameter, don’t specify --manifest or --metadata."
```

Comment:
```suggestion
      "If you specify this parameter, don’t specify --manifest or --metadata. If the comma-separated list you’re supplying contains spaces, enclose the entire comma-separated list in one set of double quotes."
```

Reply:
Added back one sentence that was in legacy command help.  I assume it's still correct?